Swiftfin/Views/VideoPlayer/Overlays/Components/ActionButtons/PlaybackSpeedActionButton.swift

```diff
@@ -30,19 +26,19 @@ extension VideoPlayer.Overlay.ActionButtons {
             Menu {
                 ForEach(PlaybackSpeed.allCases, id: \.self) { speed in
                     Button {
-                        playbackSpeed = Float(speed.rawValue)
+                        videoPlayerManager.playbackSpeed = speed
                         videoPlayerProxy.setRate(.absolute(Float(speed.rawValue)))
                     } label: {
-                        if Float(speed.rawValue) == playbackSpeed {
+                        if speed == videoPlayerManager.playbackSpeed {
                             Label(speed.displayTitle, systemImage: "checkmark")
                         } else {
                             Text(speed.displayTitle)
                         }
                     }
                 }
 
-                if !PlaybackSpeed.allCases.map(\.rawValue).contains(where: { $0 == Double(playbackSpeed) }) {
```

Comment:
I can't remember why this exists (checking for a non-defined playback speed), so we can remove it.